jlm/hls/util/view.hpp

```diff
@@ -13,17 +13,61 @@
 namespace jlm::hls
 {
 
+enum ViewColors
+{
+  NONE,
```

Comment:
That is not really a color? Why is it necessary? What is the difference between NONE and not specifying a color?

Reply:
@davidmetz
GetDefaultLabel() use "" as the default, while GetDefaultColor() use black.
What's the significance of using "" instead of a predefined color?

Reply:
NONE has been removed